src/test/java/org/opentripplanner/updater/trip/moduletests/addition/AddedTest.java

```diff
@@ -120,6 +127,65 @@ void repeatedlyAddedTripWithNewRoute() {
     assertNotNull(env.getTransitService().getRouteForId(firstRoute.getId()));
   }
 
+  @Test
+  public void addedTripWithSkippedStop() {
+    var env = RealtimeTestEnvironment.gtfs();
+    var builder = new TripUpdateBuilder(ADDED_TRIP_ID, SERVICE_DATE, ADDED, env.timeZone);
+    builder
+      .addStopTime(STOP_A1_ID, 30, DropOffPickupType.PHONE_AGENCY)
+      .addStopTime(STOP_B1_ID, 40, DropOffPickupType.COORDINATE_WITH_DRIVER, SKIPPED)
+      .addStopTime(STOP_C1_ID, 48, SKIPPED)
+      .addStopTime(STOP_D1_ID, 55);
+    var tripUpdate = builder.build();
+
+    env.applyTripUpdate(tripUpdate);
+
+    // THEN
+    final TripPattern tripPattern = assertAddedTrip(ADDED_TRIP_ID, env);
+    assertEquals(PickDrop.CALL_AGENCY, tripPattern.getBoardType(0));
+    assertEquals(PickDrop.CANCELLED, tripPattern.getAlightType(1));
+    assertEquals(PickDrop.CANCELLED, tripPattern.getBoardType(1));
+    assertEquals(PickDrop.CANCELLED, tripPattern.getAlightType(2));
+    assertEquals(PickDrop.CANCELLED, tripPattern.getBoardType(2));
+    assertEquals(PickDrop.SCHEDULED, tripPattern.getAlightType(3));
+    final TimetableSnapshot snapshot = env.getTimetableSnapshot();
+    final Timetable forToday = snapshot.resolve(tripPattern, SERVICE_DATE);
+    final int forTodayAddedTripIndex = forToday.getTripIndex(ADDED_TRIP_ID);
+    final TripTimes tripTimes = forToday.getTripTimes(forTodayAddedTripIndex);
```

Comment:
```suggestion
    var snapshot = env.getTimetableSnapshot();
    var forToday = snapshot.resolve(tripPattern, SERVICE_DATE);
    int forTodayAddedTripIndex = forToday.getTripIndex(ADDED_TRIP_ID);
    var tripTimes = forToday.getTripTimes(forTodayAddedTripIndex);
```

I know that you copied these lines from a previous version of the tests but nowadays we write them without the types so it's easier to follow.

Reply:
OK I will change the types to `var` but I still normally prefer `final var` when writing Java code, because it makes the variable immutable (such that a lack of `final` means that the variable is intended to be mutated).

What do you think about the usage of `final`?